src/rime/dict/dict_compiler.cc

```diff
@@ -230,6 +231,26 @@ bool DictCompiler::BuildPrism(const string &schema_file,
         script.clear();
       }
     }
+
+#if 0
+    // build corrector
+    bool enable_correction = false; // Avoid if initializer to comfort compilers
```

Comment:
nit: be consistent about capitalizing phrase/sentence in code comments. the previous line is not capitalized.